### DISCUSSION TITLE
Is there any user/admin documentation?

### DISCUSSION THREAD

**Discussion by mytlogos** [blackcandy-org/blackcandy#388] [Q&A] [answered]
I tried to find a website or documentation files for this app, so i can investigate if this app is for me or not, before i try it out myself.
Yes, i know there is a demo, but that is a limited experience.

As it looks right now, the only documentation is the README.md file and the upgrade docs in the `docs/` folder.
The README.md primarily describes the installation process (with config via environment variables).

What i would want to know of any self-hosted app are things like:

-  what are the features of this application
-  what authentication methods does this app (not) support
  - does it have a user/role management
- how do i use it? (user manual)
- how do i administrate it (admin manual)
- what hardware requirements does it have (cores, ram, architecture, platform)
- does it have an api
- does it have known integrations with other software/services
- does it phone back (e.g. telemetry, update-check, license)

While most of these are not necessary, it still would be nice to have them documented.

**Comment by aidewoode** [comment, marked as answer]
Thanks for your advice,  right now there is no such documentation in this project. And better documentation is the one of the goals that the project should achieve.  But writing docs is not an easy thing, it takes time.  Anyway, this project is open to any documentation contribution as well.

**Reply by mytlogos** [reply]
Thanks for clarifying, thought i missed something.
